cmd/gardener-controller-manager/app/gardener_controller_manager.go

```diff
@@ -245,6 +246,9 @@ func NewGardener(config *componentconfig.ControllerManagerConfiguration) (*Garde
 	logger := logger.NewLogger(config.LogLevel)
 	logger.Info("Starting Gardener controller manager...")
 	logger.Infof("Feature Gates: %s", gardenerfeatures.ControllerFeatureGate.String())
+	if err := flag.Lookup("v").Value.Set(fmt.Sprintf("%d", config.KubernetesLogLevel)); err != nil {
```

Comment:
We switched to only support exactly one flag for the controller-manager: `--config` (that's where the Kubernetes community is going to with componentconfig). Let's stay with that and not introduce further CLI flags.